tests/gdx-tests/src/com/badlogic/gdx/tests/TiledMapObjectPropertyTest.java

```diff
@@ -0,0 +1,190 @@
+
+package com.badlogic.gdx.tests;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.OrthographicCamera;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.graphics.glutils.ShapeRenderer;
+import com.badlogic.gdx.maps.MapLayer;
+import com.badlogic.gdx.maps.MapObject;
+import com.badlogic.gdx.maps.MapObjects;
+import com.badlogic.gdx.maps.MapProperties;
+import com.badlogic.gdx.maps.objects.RectangleMapObject;
+import com.badlogic.gdx.maps.tiled.TiledMap;
+import com.badlogic.gdx.maps.tiled.TiledMapRenderer;
+import com.badlogic.gdx.maps.tiled.TmxMapLoader;
+import com.badlogic.gdx.maps.tiled.renderers.OrthogonalTiledMapRenderer;
+import com.badlogic.gdx.math.Rectangle;
+import com.badlogic.gdx.tests.utils.GdxTest;
+import com.badlogic.gdx.tests.utils.OrthoCamController;
+import com.badlogic.gdx.utils.ObjectMap;
+import com.badlogic.gdx.utils.ObjectSet;
+import com.badlogic.gdx.utils.ScreenUtils;
+
+import java.util.Iterator;
+
+/** Test for successful loading of "object" properties for tiled objects in a map. The value of the "object" property is the id of
+ * another tiled object. This test ensures that the "object" property is correctly loaded and points to the correct object.
+ *
+ * The following should be true: - Object with id 1 should have props: - Points_To_ID_1 = 1 - Points_To_ID_2 = 2 - Points_To_ID_5
+ * = 5 - Object with id 2 should have props: - Points_To_ID_3 = 3 - Points_To_ID_4 = 4 - Object with id 3 should have props: -
+ * Points_To_ID_2 = 2 - Object with id 4 should have props: - Points_To_ID_1 = 1 - Objects with id's 5 and 6 should have props: -
+ * Placeholder = 0 */
+public class TiledMapObjectPropertyTest extends GdxTest {
+
+	private TiledMap map;
+	private SpriteBatch batch;
+	private ShapeRenderer shapeRenderer;
+	private OrthographicCamera camera;
+	private TiledMapRenderer mapRenderer;
+	private boolean error;
+
+	@Override
+	public void create () {
+		Gdx.app.log("TiledMapObjectPropertyTest", "Running test...");
+		try {
+			float w = Gdx.graphics.getWidth();
+			float h = Gdx.graphics.getHeight();
+
+			camera = new OrthographicCamera();
+			camera.setToOrtho(false, (w / h) * 512, 512);
+			camera.zoom = .5f;
+			camera.update();
+
+			OrthoCamController cameraController = new OrthoCamController(camera);
+			Gdx.input.setInputProcessor(cameraController);
+
+			Gdx.app.log("TiledMapObjectPropertyTest", "Load map...");
+			map = new TmxMapLoader().load("data/maps/tiled-objects/test-object-properties.tmx");
+			Gdx.app.log("TiledMapObjectPropertyTest", "Finish loading map!");
+
+			batch = new SpriteBatch();
+			shapeRenderer = new ShapeRenderer();
+			mapRenderer = new OrthogonalTiledMapRenderer(map);
+
+			MapObjects objects = map.getLayers().get("Objects").getObjects();
+
+			ObjectMap<Integer, MapProperties> idPropMap = new ObjectMap<>();
```

Comment:
I forgot to post that this is the other usage of ObjectMap with Integer keys, and could be an IntMap.